dot/sync/chain_sync_integeration_test.go

```diff
@@ -83,22 +45,23 @@ func TestChainSync_SetPeerHead(t *testing.T) {
 
 	// test case where peer has a lower head than us, but they are on the same chain as us
 	cs.blockState = new(syncmocks.BlockState)
-	header, err := types.NewHeader(common.NewHash([]byte{0}),
-		trie.EmptyHash, trie.EmptyHash, number, types.NewDigest())
+	header, err := types.NewHeader(common.NewHash([]byte{0}), trie.EmptyHash, trie.EmptyHash, 1000,
+		types.NewDigest())
 	require.NoError(t, err)
 	cs.blockState.(*syncmocks.BlockState).On("BestBlockHeader").Return(header, nil)
-	fin, err := types.NewHeader(common.NewHash([]byte{0}),
-		trie.EmptyHash, trie.EmptyHash, number-2, types.NewDigest())
+	fin, err := types.NewHeader(common.NewHash([]byte{0}), trie.EmptyHash, trie.EmptyHash, 998,
+		types.NewDigest())
 	require.NoError(t, err)
 	cs.blockState.(*syncmocks.BlockState).On("GetHighestFinalisedHeader").Return(fin, nil)
 	cs.blockState.(*syncmocks.BlockState).On("GetHashByNumber", mock.AnythingOfType("uint")).Return(hash, nil)
 
-	err = cs.setPeerHead(testPeer, hash, number-1)
+	number = 999
+	err = cs.setPeerHead(testPeer, hash, number)
 	require.NoError(t, err)
 	expected = &peerState{
 		who:    testPeer,
 		hash:   hash,
-		number: number - 1,
+		number: number,
```

Comment:
nit I'd keep it as it was, so it scales with the initial constant number set; same below

Reply:
Updated.